docs/customization/adding_attributes_to_users.md

```diff
@@ -0,0 +1,159 @@
+# Adding Attributes to Users
+
+If you need to add new attributes like phone numbers, employee or school IDs, etc.
+to users, one way is to add columns to `users` table.
+
+## Create Migration File
+
+Create a migration file to add new columns.
+
+You can easily create a file for it with the `spark` command:
+```console
+php spark make:migration AddMobileNumberToUsers
+```
+
+And write code to add/drop columns.
+
+```php
+<?php
+
+namespace App\Database\Migrations;
+
+use CodeIgniter\Database\Migration;
+
+class AddMobileNumberToUsers extends Migration
+{
+    public function up()
+    {
+        $fields = [
+            'mobile_number' => ['type' => 'VARCHAR', 'constraint' => '20', 'null' => true],
+        ];
+        $this->forge->addColumn('users', $fields);
```

Comment:
I think it is better to use `$tables['users']`.

Reply:
Done.